PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/webapp/WebAppComboBoxModel.java

```diff
@@ -42,13 +43,21 @@ public WebAppComboBoxModel(final ResourceEx<WebApp> resourceEx) {
     }
 
     public WebAppComboBoxModel(WebAppSettingModel webAppSettingModel) {
+        this.resourceId = webAppSettingModel.getWebAppId();
         this.appName = webAppSettingModel.getWebAppName();
         this.resourceGroup = webAppSettingModel.getResourceGroup();
         this.os = webAppSettingModel.getOS().name();
         this.runtime = webAppSettingModel.getOS() == OperatingSystem.LINUX ?
                        webAppSettingModel.getLinuxRuntime().toString() : webAppSettingModel.getWebContainer();
         this.subscriptionId = webAppSettingModel.getSubscriptionId();
-        this.isNewCreateResource = true;
+        this.isNewCreateResource = webAppSettingModel.isCreatingNew();
         this.webAppSettingModel = webAppSettingModel;
     }
+
+    public static boolean isSameWebApp(WebAppComboBoxModel first, WebAppComboBoxModel second) {
+        return StringUtils.equalsAnyIgnoreCase(first.resourceId, second.resourceId) ||
```

Comment:
resourceId is enough?

Reply:
For new create web app, we have no webapp id in confiugration, and use webapp name and resource group for validation